src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -1334,6 +1334,11 @@ public function save_ticket( $post_id, $ticket, $raw_data = array() ) {
 		// Makes sure it's an Int after this point
 		$data['stock'] = (int) $data['stock'];
 
+		// The only available value lower than zero is -1 which is unlimited
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 1d79f07492bd7ffa9a804cea838513ff</sub>

Reply:
```suggestion
		// The only available value lower than zero is -1 which is unlimited.
```